api/source/output_formats/enums.rst

```diff
@@ -82,9 +82,9 @@ Fidelity Enumeration
     <span style="padding: 20px">
     MECHANICAL</br>
     <span style="padding: 20px">
-    PREMIUM</br>
+    PREMIUM <font color="black">— The <a href="http://docs.cielo24.com/en/latest/output_formats/formats.html#job-info-format-label">Job Info Format</a> returns this value as </font>STANDARD</br>
```

Comment:
These links should be relative so that they work regardless of where the docs are deployed: `<a href="formats.html#job-info-format-label">`